disputer/test/Disputer.js

```diff
@@ -171,6 +171,10 @@ contract("Disputer.js", function(accounts) {
       { from: liquidator }
     );
 
+    // Try disputing before any mocked prices are set, simulating a situation where the pricefeed
+    // fails to return a price. The disputer should fail gracefully.
+    await disputer.queryAndDispute();
```

Comment:
Does it fail gracefully? on line 187 we check for the number of info level events and the call to `this.logger.warn` from the disputer bot should increment this number if I'm not mistaken. Did it _actually_ log this error?

Reply:
Does a `logger.warn` increment the logger? I thought it only catches info level events

Reply:
to my understanding as the logger is configured here with:
```
      transports: [new SpyTransport({ level: "info" }, { spy: spy })]
```

any event that is info or above (including `warn`) should be sent to the SpyTransport.

Reply:
Nevermind. This actually doesn't emit a warning because the `null` price is caught higher up on the `isDisputable` check